client/auth/action-creators.ts

```diff
@@ -17,7 +18,7 @@ function idRequest<
   ActionType extends Extract<IdRequestable, { type: ActionTypeName }>
 >(
   type: ActionTypeName,
-  fetcher: () => Promise<ActionType['payload']>,
+  fetcher: (dispatch: any) => Promise<ActionType['payload']>,
```

Comment:
Could use some help with typing the `dispatch` function here.

Reply:
sendVerificationEmail isn't an ID'd request (it's not associated with a form, we'd never show the error from the reducer, so why do this?), so just don't add this and don't use this function for it and you won't have this problem.

For future reference though, `dispatch-registry` exports a typing for the dispatch function.